react/features/calendar-sync/middleware.js

```diff
@@ -34,34 +42,53 @@ MiddlewareRegistry.register(store => next => action => {
     return result;
 });
 
+/**
+ * Clears the calendar access status when the app comes back from
+ * the background. This is needed as some users may never quit the
+ * app, but puts it into the background and we need to try to request
+ * for a permission as often as possible, but not annoyingly often.
+ *
+ * @private
+ * @param {Object} store - The redux store.
+ * @param {Object} action - The Redux action.
+ * @returns {void}
+ */
+function _maybeClearAccessStatus(store, action) {
+    const { appState } = action;
+
+    if (appState === 'background') {
+        const { dispatch } = store;
+
+        dispatch(updateCalendarAccessStatus(undefined));
```

Comment:
undefined is essentially an unsetting so that the app behaves like it hasn't asked for permission? I wonder if another explicit action would make that clearer or maybe it's clear enough. I'm very meek.